### DISCUSSION TITLE
Printable job sheet

### DISCUSSION THREAD

**Discussion by AndyVincentUK** [sn4k3/UVtools#406] [Ideas]
Maybe this is already there or a script exists that I have yet to find, but it would be really cool to be able to print out a 'job sheet' from UVTools. All of the information is there in the app but it would be very helpful to have an easy way to print the following:

- Filename
- Thumbnail
- Print parameters (the info along the bottom of the screen)
- Object size (px and mm)
- Perhaps also the 'I printed this' info for material 

I assume the file would likely need to be Word (or PDF) although hopefully the operating system could take care of the format (I use Win). The intent would then be to be able to drop the job sheet file into a folder with the STL, printer and original design file and avoid the need to open each file in UVTools to check the print info.

Many thanks

**Comment by sn4k3** [comment]
The ideia is good and is something i already tought to implement, but not yet move in that direction.

For now if you want that information, you can use: File -> Extract, and then delete all layer images to keep only the information there.
